lib/checkers.cpp

```diff
@@ -65,7 +65,7 @@ namespace checkers {
         {"CheckBoost::checkBoostForeachModification",""},
         {"CheckNullPointer::nullPointer",""},
         {"CheckNullPointer::nullConstantDereference",""},
-        {"CheckNullPointer::aithmetic",""},
+        {"CheckNullPointer::arithmetic",""},
```

Comment:
These literals should probably be put into a separate header so they are not duplicated in the code.